src/main/java/com/example/springbootbookshop/entity/RoleName.java

```diff
@@ -0,0 +1,7 @@
+package com.example.springbootbookshop.entity;
+
+public enum RoleName {
+    ADMIN,
+    MANAGER,
```

Comment:
you never use  role MANAGER, so let’s  remove it

Reply:
Not fixed